fuel-rxjava/src/main/kotlin/com/github/kittinunf/fuel/rx/RxFuel.kt

```diff
@@ -224,6 +225,11 @@ fun <T : Any> Request.rxObjectPair(deserializable: Deserializable<T>) = rxResult
  */
 fun <T : Any> Request.rxObjectTriple(deserializable: Deserializable<T>) = rxResultTriple(deserializable)
 
+/**
+ * Returns a reactive stream for a [Single] value with only complete(success with [Unit]) or error signal
+ */
+fun Request.rxUnit(): Single<Unit> = rxResponseSingle(EmptyDeserializer)
```

Comment:
🤔 Since we are no longer care about the emitting data? How about turning this into a `Completable`, I am not really sure whether the `Single` still fits the bill here anymore. WDYT?

Reply:
Oh.. You're right, I can't find `onSuccess` function without passing an argument in Rx, however that's the way to complete an stream in Reactor which I'm more familiar with. `Completable` should be the chosen one.